schemas/documents/shared/classPrefinanceTimelineStatus.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'classPrefinanceTimelineStatus',
```

Comment:
I've separated the class timeline statuses from the project ones since it seems like those might not really overlap